server/api/routes.md

```diff
@@ -0,0 +1,49 @@
+**USER: (/api/user)**
+
+* register guest user (POST) - occurs as soon as any user visits site (any page). Sets cookie (sessionId)
+* create user account (POST)
+* authenticate registered user (POST) - sets cookie (some version of userId)
+* retrieve single user, or all users (GET)
+* update user (PUT)
+* delete user account (DELETE)
+
+**CART/CART-ITEM: (/api/cart)**
```

Comment:
Let's have a separate section for CART and CART-ITEM models?

I suggest we might use
GET /api/cart/:cartId to return a cart and  ALL cart items
GET /api/cartItem/:itemId  to return a single cart item (this probably won't be leveraged directly by a call from the frontend)

Reply:
sure